manifest/manifest.go

```diff
@@ -147,7 +151,7 @@ func GuessMIMEType(manifest []byte) string {
 			}
 			return ociMan.Config.MediaType
 		}
-		return DockerV2Schema2MediaType
+		return imgspecv1.MediaTypeImageManifest
```

Comment:
Please document that we are guessing this is an OCI artifact, not necessarily a container image.